manifests/install/windows.pp

```diff
@@ -78,10 +97,16 @@
                               exit 0; \
                             } \
                             exit 1; }.Invoke()",
+    onlyif  => "${::system32}\\WindowsPowerShell\\v1.0\\powershell.exe \
+                  -NoProfile \
+                  -NoLogo \
+                  -NonInteractive \
+                  ${_puppet_check} ${::puppet_agent::_expected_package_version} ${_msi_location}",
```

Comment:
If the powershell script throws errors, the `exec` command will not be executed, and the errors will not show on puppet output.